### PR TITLE
style: new wizard modal for "Accounts"

### PR DESCRIPTION
**Pull request by peterpeterparker**
# Motivation

Use the new wizard modal design in "Accounts" related modals

# PRs

- [x] needs https://github.com/dfinity/gix-components/pull/82

# Changes

No functionnal / logic changes beside using `Wizard` component instead of `LegacyWizard`.

One or two description texts deleted for UX reason in accordance with Mischa.

Also removed unused `CardItem` cmp.

### CODE REVIEW

File: frontend/src/lib/themes/legacy.scss [hunk 1]
```diff
@@ -57,3 +57,20 @@ div.modal.legacy {
     --modal-legacy-big-border-radius: var(--modal-legacy-small-border-radius);
   }
 }
+
+div.modal.legacy {
```
Comment (lmuntaner) [inline frontend/src/lib/themes/legacy.scss:61]: why this?

**Review by lmuntaner — COMMENTED** [review]
No screenshots 😢 ?

**Comment by peterpeterparker** [comment]
Just look the same with the [gix.design](https://ipi2f-uqaaa-aaaad-aabza-cai.ic0.app/components/modal.html) modal so I spared my self the screenshots. if you want some, can do?

File: frontend/src/lib/themes/legacy.scss (hunk 1 above)
Comment (peterpeterparker) [reply frontend/src/lib/themes/legacy.scss:61]: there are still some legacy modal

also as described above, PR needs https://github.com/dfinity/gix-components/pull/82

Comment (lmuntaner) [reply frontend/src/lib/themes/legacy.scss:61]: ok, thanks

**Comment by lmuntaner** [comment]
> Just look the same with the [gix.design](https://ipi2f-uqaaa-aaaad-aabza-cai.ic0.app/components/modal.html) modal so I spared my self the screenshots. if you want some, can do?

I would have liked to see how they look like, but not mandatory.

**Review by lmuntaner — APPROVED** [review]

**Pull request merged**